CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 9.0.0
+
+* Log govuk request id in json in whitehall ([#300](https://github.com/alphagov/govuk_app_config/pull/300))
+* BREAKING: Remove $stdout, $stderr and $real_stdout redirections.
+* BREAKING: Change error log behaviour from logging JSON to full string
```

Comment:
```suggestion
* Log govuk request id in json in whitehall ([#300](https://github.com/alphagov/govuk_app_config/pull/300))
* BREAKING: Remove $stdout, $stderr and $real_stdout redirections ([#300](https://github.com/alphagov/govuk_app_config/pull/300))
* BREAKING: Change error log behaviour from logging JSON to full string ([#300](https://github.com/alphagov/govuk_app_config/pull/300))
```

Our convention would be to have the PR number against each change, even if it's the same PR

Reply:
We should change: "Log govuk request id in json in whitehall" as the impact will be wider than Whitehall. 

I'd suggest changing this to "Add govuk_request_id to JSON logging for apps with gds-api-adapters"